### ISSUE TITLE
Use new format of timeline to pick up Timeline events

### ISSUE THREAD

**Issue by axemclion** [axemclion/browser-perf#31]
Chrome 41 has deprecated Timeline and enableTimeline - it now gets the same events in Tracing. This is enabled by using `traceCategories: 'blink.console,disabled-by-default-devtools.timeline'` in the `perfLoggingPrefs` capabilities. 

The timeline format has also changed.

**Comment by axemclion** [comment]
Updated the TimelineMetrics using this - https://github.com/axemclion/browser-perf/pull/30

Also need to update it in ChromeTracingMetrics.

**Comment by axemclion** [comment]
Reference for [Trace event format](https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/edit#)

1. (Normalize](https://github.com/angular/angular/blob/8ccafb0524e3ac4c51af34ef88e0fe27482336a6/modules/benchpress/src/webdriver/chrome_driver_extension.js#L60) perf log

2. Process events from [Perflog](https://github.com/angular/angular/blob/8ccafb0524e3ac4c51af34ef88e0fe27482336a6/modules/benchpress/src/metric/perflog_metric.js)
Basically break down completed events (X events) and sort them, send begin and end events.

**Comment by izhutaev-sugarcrm** [comment]
Hello @axemclion, is this issue related to "result" do not contain any of TimelineMetrics (such as "FunctionCall", "EventDispatch" and e.t.c)? I've recently updated to 1.2.4 with this PR https://github.com/axemclion/browser-perf/pull/30 applies, but I can't see TimelineMetrics metrics that were before (Chrome 42).

**Comment by axemclion** [comment]
@izhutaev-sugarcrm Yes, that is correct. You should be able to see timeline if you pick up the master branch on the project - that has the code. 

I am working on refactoring the code and then putting it up on npm.

**Comment by izhutaev-sugarcrm** [comment]
@axemclion Thanks for the quick reply! I will try master branch and than update when new version will be available

**Comment by axemclion** [comment]
Published a quick fix to npm - browser-perf@1.2.5 that fixes this issue. 
Closing this issue since timeline data is available. However, note that this breaks the RuntimePerfMetrics. Will open a bug for that and fix it.

**Comment by axemclion** [comment]
Fixed in https://github.com/axemclion/browser-perf/commit/6c01a7b8c8709c8a572080f65624b1ebd353c97c
